gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 # Project
 android_compileSdk = "34"
-android_gradle_plugin = "8.2.2"
+android_gradle_plugin = "8.7.0"
```

Comment:
Does upgrading the AGP need any other changes regarding deprecated features etc.

Reply:
Nope. Mostly, all releases targets bug fixes and lint warnings.